dns.go

```diff
@@ -81,6 +84,8 @@ func (h *RR_Header) len(off int, compression map[string]struct{}) int {
 	return l
 }
 
+func (r1 *RR_Header) isDuplicate(r2 RR) bool { return false }
```

Comment:
maybe nicer to implement the code that checks the header here instead of in the IsDuplicate (in duplicate.go)?

Reply:
I had that before a3d7eba50825d546074d81084e639bbecf9cbb57, but changed it back to this approach. I’m happy to revert a3d7eba50825d546074d81084e639bbecf9cbb57 if you’d like.

Reply:
The diff would like https://github.com/miekg/dns/compare/master...tmthrgd:a3d7eba%5E if I do revert that commit.

Reply:
why did you change it back? I think here is better, but would like to know the reason you reverted that?

Reply:
@miekg Are you happy with how it is now?